src/NotepadNext/NotepadNextApplication.cpp

```diff
@@ -403,6 +409,9 @@ void NotepadNextApplication::loadTranslation(QLocale locale)
 
     // Load translation for Qt components
     //  e.g. `i18n/qt_en.qm`
+#ifdef Q_OS_LINUX
+    languagePath = QLibraryInfo::location(QLibraryInfo::TranslationsPath);
```

Comment:
QLibraryInfo::TranslationsPath should work on Windows and MacOS, but I didn't test it. Currently only used in linux.

Reply:
No problem. When I get some free time I'll test it out on Windows unless anyone else gets to it before I do :)